src/bp/core/services/cms.ts

```diff
@@ -360,6 +372,17 @@ export class CMSService implements IDisposeOnExit {
     await this.ghost.forBot(botId).upsertFile(this.elementsDir, fileName, content)
   }
 
+  private translateIfRequired(element: ContentElement, language?: string) {
+    if (!language) {
```

Comment:
by making the check in the calling func, you can remove this, remove the optionality in the language arg and rename the function to translateElement